src/checkers/inference/InferenceLauncher.java

```diff
@@ -226,7 +225,7 @@ private void removeXmArgs(List<String> argList, int preJavacOptsSize, int postJa
         String insertAnnotationsScript = pathToAfuScripts+"insert-annotations-to-source";
         if (!InferenceOptions.inPlace) {
             final File outputDir = new File(InferenceOptions.afuOutputDir);
-            TestUtilities.ensureDirectoryExists(outputDir);
```

Comment:
`org.checkerframework.framework.test` depends on JUnit, so don't use any classes in that package.

Reply:
I'm not sure I understand. Is JUnit now only a test dependency and therefore normal, non-test code shouldn't depend on it?
Should we therefore move `framework/src/org/checkerframework/framework/test` to a different location, if it has additional dependencies?

Reply:
> Is JUnit now only a test dependency and therefore normal, non-test code shouldn't depend on it?

Yes, this line: ` testCompile 'junit:junit:4.12'` means that JUnit is a compile time dependency for tests only.

>Should we therefore move framework/src/org/checkerframework/framework/test to a different location, if it has additional dependencies?

Yes.  We currently don't include the JUnit classes in checker.jar anyways.  When we change the Checker Framework build to Gradle, we should move that package to a different location.  (Where/how we can discuss when I post the Checker Framework Gradle scripts.)